packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -304,6 +305,15 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
     return null;
   }
 
+  @autobind
+  private _onRenderDescription(props: ITextFieldProps): JSX.Element | null {
```

Comment:
we've moving away from autobind to arrow functions.

Reply:
ok, I've made the change